config/jobs/kubernetes-sigs/kueue/kueue-release-blocking.yaml

```
@@ -0,0 +1,245 @@
+periodics:
+  - interval: 1d
+    name: periodic-kueue-test-unit-main
+    annotations:
+      testgrid-dashboards: sig-scheduling
+      testgrid-tab-name: periodic-kueue-test-unit-main
+      description: "Run kueue unit tests"
```

```suggestion
      description: "Run periodic kueue unit tests"
```

Just to distinguish these tests from the existing unit tests/e2e tests?